membership.go

```diff
@@ -400,7 +410,7 @@ func multicastAnnounce(addr string) error {
 
 		logfTrace("Sent announcement multicast to %v\n", fullAddr)
 
-		if GetMulticastAnnounceInterval() > 0 {
+		if GetMulticastAnnounceIntervalSeconds() > 0 {
 			time.Sleep(time.Second * 10)
```

Comment:
You probably don't mean to hard-code 10 here.

Reply:
Good find 👍